hbase-handler/pom.xml

```diff
@@ -223,8 +223,9 @@
       <scope>test</scope>
     </dependency>
     <dependency>
-      <groupId>org.eclipse.jetty</groupId>
-      <artifactId>jetty-runner</artifactId>
+      <groupId>org.apache.tomcat</groupId>
+      <artifactId>tomcat-jasper</artifactId>
+      <version>${tomcat.version}</version>
```

Comment:
why do you need to provide a version if you have a declaration in the root pom DM?